test/utility/test_prefix_sum.cpp

```diff
@@ -97,7 +97,7 @@ TEST(PrefixSum, Array) {
                                    scan_op<SRC_VAL, DST_VAL>, combiner<DST_VAL>, galois::Array>;
   PFXSUM pfxsum(arr, arr);
 
-  PANDO_CHECK(pfxsum.initialize());
+  PANDO_CHECK(pfxsum.initialize(pando::getPlaceDims().core.x * pando::getPlaceDims().core.y));
```

Comment:
I have assumed that this mix of init by num_cores vs num_hosts has been initentional

Reply:
yes.